_apps/reviewed.md

```diff
@@ -0,0 +1,31 @@
+---
+# A human-friendly name of your listing
```

Comment:
These comments still need to be removed

Reply:
@hiimbex Oops. I missed. Thank you for you review. @g-ozawa Please remove the comment.